README.md

```diff
@@ -1,10 +1,10 @@
 # AQUA-KV
 
-This is the supplementary code for Cache Me If You Must: Adaptive Key-Value Quantization for Large Language Models
-
-The current code version is designed for reproducing and prototyping. Efficient inference kernels are TBU.
+[![arXiv](https://img.shields.io/badge/arXiv-2501.19392-b31b1b.svg)](https://arxiv.org/abs/2501.19392)
```

Comment:
Not related to the PR

Reply:
kk; removed it

Reply:
not entirely, delta is still non-trivial